src/validation.cairo

```diff
@@ -154,6 +152,13 @@ fn next_prev_timestamps(self: @ChainState, block: @Block) -> Span<u32> {
     timestamps.span()
 }
 
+fn compute_block_height(block_height: Option<u32>) -> Option<u32> {
```

Comment:
Let's call it next_block_height, similar to next_prev_timiestamps (also please add docstring)